### DISCUSSION TITLE
I am not able to connect to github pages or able to perform git operation like cloning and pushing using git cli on all my devices

### DISCUSSION THREAD

**Discussion by kartikeysaini** [community/community#110993] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i am getting connection timeout on the website and when i try to ping github.com it sometime work but most of the time it shows connection timeout. Also while cloning a repo its getting stuck at receiving objects.
![Uploading Screenshot 2024-03-05 201500.png…]()

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @kartikeysaini! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, are you still seeing this issue on your end? If so, then reopening this discussion post would help with visibility. 

I've gone ahead and moved this to the correct category for you. Good luck!
